diff --git a/common/lowram/device-common-1gb.mk b/common/lowram/device-common-1gb.mk
index 4aa2046..0211c66 100644
--- a/common/lowram/device-common-1gb.mk
+++ b/common/lowram/device-common-1gb.mk
@@ -13,7 +13,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_PROPERTY_OVERRIDES += ro.config.low_ram=true
 
 PRODUCT_COPY_FILES += \
-    $(LOCAL_PATH)/init.lowram.rc:$(TARGET_COPY_OUT_VENDOR)/etc/init/init.lowram.rc \
+    device/glodroid/common/lowram/init.lowram.rc:$(TARGET_COPY_OUT_VENDOR)/etc/init/init.lowram.rc \
 
 LOCAL_PATH := device/glodroid/common
 include $(LOCAL_PATH)/device-common.mk
