diff --git a/common/ueventd.common.rc b/common/ueventd.common.rc
--- a/common/ueventd.common.rc
+++ b/common/ueventd.common.rc
@@ -25,3 +25,6 @@
 
 /dev/dma_heap/system                                   0666    system       graphics
 /dev/dma_heap/linux,cma                                0666    system       graphics
+
+# HCI rfkill index is dynamic; limit writes to Bluetooth controllers.
+/sys/devices/*/bluetooth/hci*/rfkill* state 0660 bluetooth net_bt no_fnm_pathname
