diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 55047df..c264bed 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -414,3 +414,8 @@ PRODUCT_COPY_FILES += $(call add-to-product-copy-files-if-exists,\
     frameworks/base/config/dirty-image-objects:system/etc/dirty-image-objects)
 
 $(call inherit-product, $(SRC_TARGET_DIR)/product/runtime_libart.mk)
+
+# Also excluded in handheld_system.mk; base_system is inherited separately.
+ifeq ($(TARGET_PRODUCT),wukongpi)
+PRODUCT_PACKAGES := $(filter-out SecureElement,$(PRODUCT_PACKAGES))
+endif
